code/modules/projectiles/projectile/bullets.dm

```diff
@@ -161,6 +161,49 @@
 	penetrating = 5
 	hitscan = 1 //so the PTR isn't useless as a sniper weapon
 
+/obj/item/projectile/bullet/rifle/tranq
+	name = "dart"
+	icon_state = "dart"
+	damage = 5
+	stun = 0
+	weaken = 0
+	drowsy = 0
+	eyeblur = 0
+	damage_type = TOX
+	step_delay = 0.25
+
+/obj/item/projectile/bullet/rifle/tranq/on_hit(var/atom/target, var/blocked = 0, var/def_zone = null)
+	var/mob/living/L = target
+	if(!(isanimal(target)))
+		if(!(isipc(target)))
+			if(!isrobot(target))
+				L.apply_effect(5, DROWSY, 0)
+				if(def_zone == "torso")
+					if(blocked < 2 && !(blocked < 1))
+						target.visible_message("<b>[target]</b> yawns.")
+					if(blocked < 1)
+						spawn(120)
+							L.apply_effect(10, PARALYZE, 0)
+							target.visible_message("<b>[target]</b> moans.")
+				if(def_zone == "head" && blocked < 2)
+					spawn(35)
+						L.apply_effect(20, PARALYZE, 0)
+				if(def_zone != "torso" && def_zone != "head")
+					if(blocked < 2 && !(blocked < 1))
+						target.visible_message("<b>[target]</b> yawns.")
+					if(blocked < 1)
+						spawn(45)
+							L.apply_effect(15, PARALYZE, 0)
+							target.visible_message("<b>[target]</b> moans.")
+	if(isanimal(target))
+		target.visible_message("<b>[target]</b> twitches, foaming at the mouth.")
+		L.apply_damage(35, TOX) //temporary until simple_mob paralysis actually works.

```

Comment:
this will just kill the mob.
Why not fix paralysis? or add some disabling effect to them.

Have you tested with stunned or weakened as well? How about with modifying stance